silkworm/silkrpc/json/transaction.cpp

```diff
@@ -53,6 +53,7 @@ void to_json(nlohmann::json& json, const Transaction& transaction) {
         json["chainId"] = rpc::to_quantity(*transaction.chain_id);
         json["v"] = rpc::to_quantity(uint64_t(transaction.odd_y_parity));
         json["accessList"] = transaction.access_list;  // EIP2930
+        json["yParity"] = rpc::to_quantity(transaction.odd_y_parity);
```

Comment:
I propose to make this interface change later.
This change can have impact in many integration test see tests/integration/goerly/....
We need agree rpcademon/erigon to be used that support this new field and updates all the integration test impacted.

Reply:
This field was introduced in EIP-1559 and is mandatory, according to the [spec](https://github.com/ethereum/execution-apis/blob/main/src/schemas/transaction.yaml#L177). Geth has already implemented it, but Erigon seems to be lagging behind.
So I can revert this change and disable unit tests, or we can fix integration tests. It seems easy enough, just updating json files, I can create another PR. It's your call!

Reply:
I really understand the point raised by @lupin012 about interface change, especially in terms of running integration tests against Erigon RpcDaemon for comparison purposes. Nevertheless, I appreciate the great value added by being compliant with the correct behaviour according to the spec, so I suggest the following:
- merging this PR as it is
- adding a new command-line option `--erigon-compatibility` to `silkrpcdaemon` to force behaviour equal to Erigon, so we can still run the integration tests for comparison